client/tooltool.py

```diff
@@ -703,6 +708,216 @@ def touch(f):
         log.warn('impossible to update utime of file %s' % f)
 
 
+def retrier(attempts=5, sleeptime=10, max_sleeptime=300, sleepscale=1.5, jitter=1):
+    """
```

Comment:
I would remove the doctring here and just keep a single line to say that this function originates from redo X.Y